File: app/src/main/java/com/petnbu/petnbu/SharedPrefUtil.java
package com.petnbu.petnbu;

import android.content.Context;
import android.content.SharedPreferences;

public class SharedPrefUtil {

    public static final String KEY_LOGGED_USER_ID = "key-logged-user-id";

    public static void saveUserId(String userId){
        SharedPreferences setting = PetApplication.getAppComponent().getSharedPref();
        setting.edit().putString(KEY_LOGGED_USER_ID, userId).apply();
    }

    public static String getUserId(Context context){
        SharedPreferences setting = PetApplication.getAppComponent().getSharedPref();
        return setting.getString(KEY_LOGGED_USER_ID, "");
    }
}

File: app/src/main/java/com/petnbu/petnbu/SharedPrefUtil.kt
package com.petnbu.petnbu

object SharedPrefUtil {

    private const val KEY_LOGGED_USER_ID = "key-logged-user-id"

    val userId: String
        get() {
            return PetApplication.appComponent?.sharedPref?.getString(KEY_LOGGED_USER_ID, "") ?: ""
        }

    fun saveUserId(userId: String) {
        PetApplication.appComponent?.sharedPref?.edit()?.putString(KEY_LOGGED_USER_ID, userId)?.apply()
    }
}
